filament/src/components/LightManager.h

```diff
@@ -244,7 +244,7 @@ class FLightManager : public LightManager {
         struct Proxy {
             // all of this gets inlined
             UTILS_ALWAYS_INLINE
-            constexpr Proxy(Base& sim, utils::EntityInstanceBase::Type i) noexcept
+            /* constexpr */ Proxy(Base& sim, utils::EntityInstanceBase::Type i) noexcept
```

Comment:
If `constexpr` isn't correct, just remove it, don't comment it. Same for the instances below.

Reply:
@romainguy, done.